proof/refine/ARM/Syscall_R.thy

```diff
@@ -1314,14 +1363,33 @@ lemma hinv_invs'[wp]:
                         dest: st_tcb_at_idle_thread')+
   done
 
+(* NOTE: This is a good candidate for corressimp at some point. For now there are some missing
+         lemmas regarding corresK and liftM. *)
```

Comment:
This is a really good comment! It explicitly tells future us what we need to do here, and it doesn't require any implicit information to be understood.